src/sentry/debug_files/artifact_bundle_indexing.py

```diff
@@ -341,7 +342,20 @@ def to_json(self) -> str:
             "files_by_debug_id": self._files_by_debug_id,
         }
 
-        return json.dumps(json_idx)
+        json_index = json.dumps(json_idx)
+
+        if len(self._files_by_url) == 0:
+            metrics.incr(
+                "artifact_bundle_flat_file_indexing.debug_id_index.size_in_mb",
+                amount=size_in_mb(json_index),
```

Comment:
I believe we have an own metrics type for this histogram-type data. Using a counter for this does not make sense. We would want to know p50, p95, etc…

Reply:
How do we collect that?

Reply:
`metrics.timing`, I know its a horrible name but it is what it is. Being used for histogram-like data all over the place:

https://github.com/getsentry/sentry/blob/b803cbba8c8de912da92f33d49a3eebd61f359db/src/sentry/models/files/abstractfile.py#L301

Reply:
Thanks for the heads up, that is a nice cache